sql/src/main/java/io/crate/executor/task/DDLTask.java

```diff
@@ -48,14 +48,18 @@ public DDLTask(UUID jobId, DDLStatementDispatcher ddlStatementDispatcher, Analyz
 
     @Override
     public void execute(final RowReceiver rowReceiver, Row parameters) {
-        ListenableFuture<Long> future = ddlStatementDispatcher.dispatch(analyzedStatement, jobId());
-        Futures.addCallback(future, new OneRowActionListener<>(rowReceiver, new Function<Long, Row>() {
+        CompletableFuture<Long> future = ddlStatementDispatcher.dispatch(analyzedStatement, jobId());
+
+        OneRowActionListener<Long> responseOneRowActionListener = new OneRowActionListener<>(rowReceiver, new Function<Long, Row>() {
```

Comment:
I think OneRowActionListener could implement `BiConsumer<T, Throwable>` directly to keep the old pattern